.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules/
 coverage/
 lib/
 *.log
+package-lock.json
```

Comment:
I assume you're adding this b/c you've used `npm`? I would advise against that, as it'll mean you're getting different versions of packages than CI, since `npm` won't consume (nor update!) the `yarn.lock`.

Eitherway, I'd prefer not to have this in the `.gitignore`, as I think it's confusing.

Reply:
okay